main.lua

```diff
@@ -20,7 +19,7 @@ local UpdateEntityAnimation = require('systems/UpdateEntityAnimation')
 local SpaceFriction = require('systems/SpaceFriction')
 
 love.load = function()
-    love.window.setMode(800, 600)
+    love.window.updateMode(State.camera.window_width, State.camera.window_height)
```

Comment:
Add `love.window.updateMode` to the mocks.

Reply:
Done